redis/redis.go

```diff
@@ -2,14 +2,15 @@
 package redis
 
 import (
+	"time"
+
 	"github.com/garyburd/redigo/redis"
-	"github.com/gregjones/httpcache"
 )
 
 // cache is an implementation of httpcache.Cache that caches responses in a
 // redis server.
-type cache struct {
-	redis.Conn
+type Cache struct {
```

Comment:
Why did you decide to make this type public?

Is it so that users can access the value of `*redis.Pool` afterwards?

You forgot to update the comment:

```
// cache is an implementation ...
```

It should become:

```
// Cache is an implementation ...
```